.github/workflows/functions.yml

```diff
@@ -113,7 +113,8 @@ jobs:
         target: [ios, tvos, macos]
         flags: [
           '--use-static-frameworks',
-          '--use-libraries'
+          '--use-libraries',
```

Comment:
Should be `--use-libraries --skip-tests` instead of creating another matrix element.